src/main/java/apoc/load/Jdbc.java

```diff
@@ -39,13 +39,9 @@
     @Context
     public Log log;
 
-    private static Connection getConnection(String jdbcUrl) throws Exception {
-        URI uri = new URI(jdbcUrl.substring("jdbc:".length()));
-        String userInfo = uri.getUserInfo();
-        if (userInfo != null) {
-            String[] user = userInfo.split(":");
-            String cleanUrl = jdbcUrl.substring(0,jdbcUrl.indexOf("://")+3)+jdbcUrl.substring(jdbcUrl.indexOf("@")+1);
-            return DriverManager.getConnection(cleanUrl, user[0], user[1]);
+    private static Connection getConnection(String jdbcUrl, LoadJdbcConfig config) throws Exception {
```

Comment:
We cannot change it directly. We have to keep the current way for at least one version and deprecated it.
So if there is no credentials passed in config we have to keep the existing approach for the time being.